frontend/tests/controllers/CollectionListTest.php

```diff
@@ -119,8 +119,8 @@ public function testCollectionAuthors() {
                 \OmegaUp\Test\Factories\QualityNomination::createSuggestion(
                     $identities[$i],
                     $problems[$j]['request']['problem_alias'],
-                    1,
-                    4,
+                    1, /* difficulty */
+                    4, /* quality */
```

Comment:
nit:

```suggestion
                    /*$difficulty=*/1,
                    /*$quality=*/4,
```

esta es la convención que usamos en otros lados.